### ISSUE TITLE
Meteor version detection

### ISSUE THREAD

**Issue by T-Grave** [CyCoreSystems/docker-meteor#38]
When my container starts I see it says: Checking Meteor version.

It reports 1.3.3 while I am 100% sure I'm using Meteor 1.4.2.1.

**Comment by ANOB-AlarmTILT** [comment]
I had this probleme earlier. Did you made the full upgrade as requested in Meteor with babel and others stuff (https://forums.meteor.com/t/meteor-1-4-2-1-is-an-important-patch-for-1-4-2-users/31190)
I used src and not bundle. And I do a reset before putting the content of my src folder.

**Comment by T-Grave** [comment]
Yeah I did the full upgrade, I am using the bundle however because I want to be able to deploy as quick as possible. (building the entire meteor app in the container on start is not as fast as a prebuilt bundle)

Two of my services are actually starting / working without any issues I can see with the "wrong" version number, I don't think it's really a big issue, just something I noticed. I do have one service that will crash the container when "Starting meteor application" (I think it has something todo with bcrypt since it's the only service using this node package) that reports the wrong version number too. But I don't think it's because of this wrong version number.
